expression/builtin_math_vec_test.go

```diff
@@ -25,6 +25,9 @@ var vecBuiltinMathCases = map[string][]vecExprBenchCase{
 	ast.Log10: {
 		{types.ETReal, []types.EvalType{types.ETReal}, nil},
 	},
+	ast.Log2: {
```

Comment:
Almost LGTM, and please post the benchmark result into your PR description.

Reply:
according to benchmark, about 2 times faster than before

```
goos: darwin
goarch: amd64
pkg: github.com/pingcap/tidb/expression
BenchmarkVectorizedBuiltinMathFunc/builtinLog2Sig-VecBuiltinFunc-8                100000             14679 ns/op               0 B/op          0 allocs/op
BenchmarkVectorizedBuiltinMathFunc/builtinLog2Sig-NonVecBuiltinFunc-8              50000             29524 ns/op               0 B/op          0 allocs/op
PASS
ok      github.com/pingcap/tidb/expression      14.005s
```